cmd/autoconnector.go

```diff
@@ -0,0 +1,310 @@
+// Copyright 2020-2021 JackTrip Labs, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/xthexder/go-jack"
+)
+
+const (
+	// CHANNELS is the default number of channels per client
+	CHANNELS = 2
+	// JT_* are pattern-matching tokens used to discover JACK ports
+	JT_RECEIVE    = ":receive_"
```

Comment:
stylewise, go doesn't like all caps or `_` in variable names (https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps). If you run `go fmt`, these vars will be flagged.